test/controllers/due.diligence.controller.spec.ts

```diff
@@ -180,6 +183,24 @@ describe("DUE_DILIGENCE controller", () => {
       expect(resp.text).toContain(WHO_IS_MAKING_FILING_URL);
     });
 
+    test(`renders the ${DUE_DILIGENCE_PAGE} page on GET method with correct back link url when feature flag is off`, async () => {
```

Comment:
can you say what feature flag is off and on in the test descriptions

Reply:
Test descriptions updated.